src/data/screenshots.json

```diff
@@ -286,257 +286,257 @@
-                    { "filename": "/assets/screenshots/2-21/en/ios/iPhone 13-traceLocation_005_categories", "alt": "Check In image 2 of 8" },
-                    { "filename": "/assets/screenshots/2-21/en/ios/iPhone 13-traceLocation_006_createQRCodeInputScreen", "alt": "Check In image 3 of 8" },
-                    { "filename": "/assets/screenshots/2-21/en/ios/iPhone 13-traceLocation_007_traceLocationOverview", "alt": "Check In image 4 of 8" },
-                    { "filename": "/assets/screenshots/2-21/en/ios/iPhone 13-traceLocation_010_checkinScreen", "alt": "Check In image 5 of 8" },
-                    { "filename": "/assets/screenshots/2-21/en/ios/iPhone 13-traceLocation_011_myQRCodes", "alt": "Check In image 6 of 8" },
-                    { "filename": "/assets/screenshots/2-21/en/ios/iPhone 13-traceLocation_008_traceLocationDetailScreen", "alt": "Check In image 7 of 8" },
-                    { "filename": "/assets/screenshots/2-21/en/ios/iPhone 13-traceLocation_009_traceLocationPdfScreen", "alt": "Check In image 8 of 8" }
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-traceLocation_004_emptyList", "alt": "Check In image 1 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-traceLocation_005_categories", "alt": "Check In image 2 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-traceLocation_006_createQRCodeInputScreen", "alt": "Check In image 3 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-traceLocation_007_traceLocationOverview", "alt": "Check In image 4 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-traceLocation_010_checkinScreen", "alt": "Check In image 5 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-traceLocation_011_myQRCodes", "alt": "Check In image 6 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-traceLocation_008_traceLocationDetailScreen", "alt": "Check In image 7 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-traceLocation_009_traceLocationPdfScreen", "alt": "Check In image 8 of 8" }
                 ]
             },
             {
                 "title": "Statistics",
                 "anchor": "ios_stats",
                 "description": "In this section you will find screenshots of the statistics. If you scroll down on the \"Status\" page, you will see \"Add local 7-day incidence\". Now swipe to the right to read the current numbers. The numbers available are the 7-day incidence, COVID-19 Patients in Intensive Care Units, confirmed new infections, warnings by app users, 7-day R-value, people who have received at least one vaccine dose, people who have basic immunization, people who have received a booster vaccination and the vaccine doeses administered.",
```

Comment:
doeses => doses

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/2785/commits/3e69001bd580758463a610bf9e1dcf7148f2a277, thanks!